src/components/MenuBars/MenuItem/MenuItem.scss

```diff
@@ -161,14 +158,14 @@ $icon-background-color-dark: #333948;
 [theme="dark"] {
   .tooltip__text {
     color: $color-white !important;
-    background-color: $tooltip-background-color-dark !important;
+    background-color: $tooltip-background-color--dark !important;
   }
 }
 
 .menu-item__icon {
```

Comment:
https://user-images.githubusercontent.com/35272402/181040213-58c8c8f9-e49f-493a-9f51-3705c0e6eebe.mov

The icons for focus mode, ready, and raised hand don't change back from the cross to the initial icon as fast as the background changes, which feels weird imo. Is that fixable?

Reply:
There was rogue property of `transition: visibility 150ms linear;` that got overwritten on larger screens but was active on smaller ones. Now it works :)